python/lsst/afw/image/basicUtils.py

```diff
@@ -94,8 +95,8 @@ def _compareWcsOverBBox(wcs0, wcs1, bbox, maxDiffSky=0.01*afwGeom.arcseconds,
     xList = numpy.linspace(bboxd.getMinX(), bboxd.getMaxX(), nx)
     yList = numpy.linspace(bboxd.getMinY(), bboxd.getMaxY(), ny)
     # we don't care about measured error unless it is too large, so initialize to max allowed
-    measDiffSky = (maxDiffSky, "?") # (sky diff, pix pos)
-    measDiffPix = (maxDiffPix, "?") # (pix diff, sky pos)
+    measDiffSky = (maxDiffSky, "?")  # (sky diff, pix pos)
+    measDiffPix = (maxDiffPix, "?")  # (pix diff, sky pos)
```

Comment:
sense you have formatting changes these should probably go on one of those commits. I wont be too strict on this, if there is only one change here or there you don't have to recraft all your commits, but please keep this in mind.